backend/editor/entries.py

```diff
@@ -21,6 +21,91 @@ def shutdown_db():
     session.close()
     driver.close()
 
+def get_label(id):
+    """
+    Helper function for getting the label for a given id
+    """
+    if (id.startswith('stopword')): return 'STOPWORDS'
+    elif (id.startswith('synonym')): return 'SYNONYMS'
+    elif (id.startswith('__header__') or id.startswith('__footer__')): return 'TEXT'
+    else: return 'ENTRY'
+
+def create_node(label, entry, mainLanguageCode):
+    """
+    Helper function used for creating a node with given id and label
+    """
+    query = [f"""CREATE (n:{label})\n"""]
+
+    # Build all basic keys of a node
+    if (label == "ENTRY"):
+        canonicalTag = entry[3:]
+        query.append(f""" SET n.main_language = "{mainLanguageCode}" """) # Required for only an entry

```

Comment:
We should use parameters as much as possible.
So here you should build a params dict and put mainLanguageCode inside. 
```python
query.append(f""" SET n.main_language = $main_language_code """)
params["main_language_code"] = main_language_code
```

Reply:
same for canonical tag below.

Also we can't avoid the mainLanguageCode insertion in "n.tags_{mainLanguageCode}"